webrtc/videoroom/lib/videoroom/pipeline.ex

```diff
@@ -274,6 +320,19 @@ defmodule VideoRoom.Pipeline do
     []
   end
 
+  defp parse_max_display_num(max_display_num_raw) do
+    case Integer.parse(max_display_num_raw) do
```

Comment:
This can be probably removed. Just put `String.to_integer()` inside `runtime.exs` and it will crash the application on invalid config value.